### ISSUE TITLE
Sleep + Wake on Y540, Catalina

### ISSUE THREAD

**Issue by inabhi9** [xiaoMGitHub/LEGION_Y7000Series_Hackintosh_Clover#39]
I'm facing sleep/wake in my Y540 (i5-9300H). It successfully wakes up from sleep 1-2 times and then crashes afterwards.

Is there any patch/clover config that I need to apply?

Thanks.

**Comment by xiaoMGitHub** [comment]
Please describe your computer configuration

**Comment by inabhi9** [comment]
Config:
Laptop Model: Lenovo Legion Y540-15IRH-PG0
CPU: i5-9300H
Graphics: Intel UHD 630, GTX 1650
RAM: 24GB
NVME: WDC PC SN720

Other notes:
- Didn't replace built-in Wi-fi card yet. Wireless device is disabled in BIOS
- Disabled Always-On USB
- Using MacBook Pro 15,1 SMBIOS
- pmset config
```
System-wide power settings:
Currently in use:
 standby              0
 womp                 0
 halfdim              1
 hibernatefile        /var/vm/sleepimage
 proximitywake        1
 powernap             0
 gpuswitch            2
 networkoversleep     0
 disksleep            10
 standbydelayhigh     86400
 sleep                0
 hibernatemode        0
 ttyskeepawake        1
 displaysleep         10
 tcpkeepalive         1
 highstandbythreshold 50
 standbydelaylow      10800
```

Let me know if you need other information

**Comment by xiaoMGitHub** [comment]
Provide your PR file, refer to [this](https://www.tonymacx86.com/threads/tool-generate-proper-problem-reporting-files.235953/)

**Comment by inabhi9** [comment]
Here: 
[debug_22541.zip](https://github.com/xiaoMGitHub/LEGION_Y7000Series_Hackintosh_Clover/files/3836339/debug_22541.zip)

**Comment by xiaoMGitHub** [comment]
I suggest you remove the useless hot patch and kext

**Comment by inabhi9** [comment]
I've removed those patches and kext. Will let you know how it goes

**Comment by inabhi9** [comment]
OK so I removed unnecessary patches and kexts though there was same issue. Later I added `darkwake=no lilucpu=16` as boot args and other couple of changes in config.plist. Now it seems working fine. So far completed 8 sleep cycles since yesterday; couple of shorts and one long (8hrs+). Although I'm still testing.

Here is my [CLOVER.zip](https://github.com/xiaoMGitHub/LEGION_Y7000Series_Hackintosh_Clover/files/3849680/CLOVER.zip). Don't know if some of those changes in config.plist are necessary
